app/services/folio_client.rb

```diff
@@ -150,12 +150,14 @@ def cancel_hold_request(user_id, hold_id)
   # TODO: after FOLIO launch rename this method to reflect service point terminology (maybe change_pickup_service_point)
   #
   # Change hold request service point
-  # @example client.change_pickup_library(hold_id: '4a64eccd-3e44-4bb0-a0f7-9b4c487abf61',
+  # @example client.change_pickup_library(resource: '4a64eccd-3e44-4bb0-a0f7-9b4c487abf61',
+  #                                        _item_key: '4a64eccd-3e44-4bb0-a0f7-9b4c487abf61',
   #                                        pickup_location_id: 'bd5fd8d9-72f3-4532-b68c-4db88063d16b')
-  # @param [String] hold_id the UUID of the FOLIO hold
+  # @param [String] resource the UUID of the FOLIO hold
+  # @param [String] _item_key this was the item key in Symphony
   # @param [String] service_point the UUID of the new service point
-  def change_pickup_library(hold_id, service_point)
-    update_request(hold_id, { 'pickupServicePointId' => service_point })
+  def change_pickup_library(resource, _item_key, service_point)
+    update_request(resource, { 'pickupServicePointId' => service_point })
   end
```

Comment:
would it make sense to change the name of this method in this PR as noted in the comments, and also get rid of the (now totally unused) `item_key`?

Reply:
I want to handle any refactoring around method names and arguments in future work. This PR is big enough and I think it'll be easier to review the refactoring work if it's approached in small chunks of work.